sparse/src/KokkosSparse_par_ilut_handle.hpp

```diff
@@ -111,26 +109,25 @@ class PAR_ILUTHandle {
 
  public:
   PAR_ILUTHandle(const size_type nrows_, const size_type nnzL_ = 0,
-                 const size_type nnzU_ = 0, const size_type max_iter_ = 1,
```

Comment:
Minor refactor. It's not nice dealing with long lists of default arguments without the ability to use named arguments like in python.